antismash/detection/hmm_detection/cluster_rules/strict.txt

```diff
@@ -232,35 +268,41 @@ RULE LAP
 
 RULE lassopeptide
     CATEGORY RiPP
+    DESCRIPTION Lasso peptide
     CUTOFF 10
     NEIGHBOURHOOD 10
     CONDITIONS PF13471 and Asn_synthase or micJ25 or mcjC
 
 RULE sactipeptide
     CATEGORY RiPP
+    DESCRIPTION Sactipeptide
     CUTOFF 20
     NEIGHBOURHOOD 10
     CONDITIONS subtilosin or thuricin or TIGR04404 or
                Subtilosin_A or skfc
 
 RULE bottromycin
     CATEGORY RiPP
+    DESCRIPTION Bottromycin-like RiPPs
     CUTOFF 20
     NEIGHBOURHOOD 10
     CONDITIONS botH
 
 RULE microviridin
     CATEGORY RiPP
+    DESCRIPTION Microviridins
     CUTOFF 20
     NEIGHBOURHOOD 10
     CONDITIONS mvd or mvnA
 
 RULE proteusin
     CATEGORY RiPP
-    COMMENT Highly modified large leader peptide RiPP class containing
-            polytheonamide (JX456532.1) and landornamide (NZ_KB235904.1:469950-493649)
-            Unfortunately, there is no real class-defining enzyme, but all
-            known precursors are similar to a nitrile hydratase
+    DESCRIPTION Highly modified large leader peptide RiPPs
+            # like polytheonamide (JX456532.1) and landornamide (NZ_KB235904.1:469950-493649)
+            # Unfortunately, there is no real class-defining enzyme, but all
+            # known precursors are similar to a nitrile hydratase
+    EXAMPLE NCBI JX456532.1 5499-20549 polytheonamide
+    EXAMPLE NCBI NZ_KB235904.1 469950-493649 landornamide
```

Comment:
The first `#` comment is duplicated in the EXAMPLE lines and the indent from the original comment was kept